src/App.tsx

```diff
@@ -81,7 +81,7 @@ const App: React.FC = () => {
   );
 
   function updateCycles(newCycles: Cycle[]) {
-    const maxOfCycles = 6;
+    const maxOfCycles = 7;
```

Comment:
You dont need the check below and May just slice array by fixed max length

Reply:
Fixed